src/Common/PlatformLinux.inl

```diff
@@ -1087,7 +1089,7 @@ typedef void *HIC;
 #define D3DTSS_TCI_CAMERASPACEREFLECTIONVECTOR    0x30000
 #define D3DTSS_TCI_SPHEREMAP                      0x40000
 
-inline BOOL SwitchToThread() { return (0 == pthread_yield()); }
+inline BOOL SwitchToThread() { return (0 == sched_yield()); }
```

Comment:
Can you explain why you change from `pthread_` function to a function from `sched.h`?

Reply:
sched_yield is more portable, same signature whereas pthread_yield returns void in another platform for example. This PR is to be able to port to a bit more oses/platforms.